molgenis-app/src/main/java/org/molgenis/rdconnect/IdCardOrganization.java

```diff
@@ -0,0 +1,49 @@
+package org.molgenis.rdconnect;
+
```

Comment:
Value objects don't have setters so that'd depend on if the setters are being used.